src/components/search/tests/MobileFilterMenu.test.jsx

```diff
@@ -6,13 +6,20 @@ import { MobileFilterMenuBase } from '../MobileFilterMenu';
 
 import { SUBJECTS, AVAILABLILITY, FACET_ATTRIBUTES } from '../data/tests/constants';
 import { renderWithRouter } from '../../../utils/tests';
+import { SearchContext } from '../SearchContext';
+
+const MobileFilterMenuWrapper = (items, searchContext = { dispatch: jest.fn }) => (
```

Comment:
nit: Might recommend treating `MobileFilterMenuWrapper` more like a component with props over a regular ol' function:

```jsx
const MobileFilterMenuWrapper = ({ items, searchContext =. dispatch: jest.fn }) => (
  // ...
);
```

Reply:
Yeah, I was having some brain fuzz when I wrote this, I think, and couldn't quite wrap my head around it. :P
But actually, I think we can just switch to using SearchData as the wrapper and call it a day.